feat(databases): connect multiple apps on a database (#2248)
Scalingo/documentation: src/_posts/platform/databases/2000-01-01-access.md

Then you connect on `localhost:10000` to reach your Scalingo database. (You'll still need to authenticate to the
database with the credential you can get from the connection string)

> ### Access a Database on Multiple Applications

Reviewer: ```suggestion
### Access the Same Database From Multiple Applications
```

If you need to share a database addon between multiple applications, you can do it by adding an environment variable in every application wanting to communicate with the database.

> In the application containing the database, you should have an environment variable containing the connection string needed to connect to the database. For a Postgresql addon, it should look like `SCALINGO_POSTGRESQL_URL`. It contains your database address, as well for its credentials. Keep this value secret!

Reviewer 2: ```suggestion
In the application containing the database, you should have an environment variable containing the connection string needed to connect to the database. For a PostgreSQL addon, it should look like `SCALINGO_POSTGRESQL_URL`. It contains your database address, as well as its credentials. Keep this value secret!
```

Author: Thank you, fixed it

> Copy this value, and in the application that needs to access it, you can create a new environment variable, and give it the value you just copied. After a restart, your app will have the connection address it needs.

Reviewer: ```suggestion
Copy this value, and in the application that needs to access it, create a new environment variable, and set it to the value you just copied. After a restart, your app will have the connection address it needs.
```

Reviewer: I think it's a bit more complex: the duplicated environment var probably has to be named in a specific way (if I name it `MY_DB`, the app/framework probably won't pick it up).
Also some frameworks/apps don't use the whole URL has connection string :-/

Author: I took your suggestion, but in any ways, when creating a new database on Scalingo: the environment variable is always the same. It's up to the user to adapt it.
